io_scene_godot/converters/grease.py

```diff
@@ -0,0 +1,367 @@
+"""Exports a grease pencil object"""
+import logging
+import bpy
+import mathutils
+
+from .material import export_material
+from ..structures import (
+    Array, NodeTemplate, InternalResource, Map, gamma_correct)
+from .utils import MeshConverter, MeshResourceKey
+from .physics import has_physics, export_physics_properties
+
+MAX_BONE_PER_VERTEX = 4
+
+def export_grease_node(escn_file, export_settings, obj, parent_gd_node):
+    """Exports a MeshInstance. If the mesh is not already exported, it will
+    trigger the export of that mesh"""
+    # If this mesh object has physics properties, we need to export them first
+    # because they need to be higher in the scene-tree
+
+    if has_physics(obj):
+        parent_gd_node = export_physics_properties(
+            escn_file, export_settings, obj, parent_gd_node
+        )
+        # skip wire mesh which is used as collision mesh
+        if obj.display_type == "WIRE":
+            return parent_gd_node
+
+    mesh_node = NodeTemplate(obj.name, "MeshInstance", parent_gd_node)
+    mesh_exporter = ArrayGreaseResourceExporter(obj)
+    mesh_id = mesh_exporter.export_grease(escn_file, export_settings)
+
+    if mesh_id is not None:
+        mesh_node['mesh'] = "SubResource({})".format(mesh_id)
+        mesh_node['visible'] = obj.visible_get()
+
+        #mesh_resource = escn_file.internal_resources[mesh_id - 1]
+        #export_object_link_material(
+        #    escn_file, export_settings, obj, mesh_resource, mesh_node
+        #)
+
+    # Transform of rigid mesh is moved up to its collision
+    # shapes.
+    if has_physics(obj):
+        mesh_node['transform'] = mathutils.Matrix.Identity(4)
+    else:
+        mesh_node['transform'] = obj.matrix_local
+
+    escn_file.add_node(mesh_node)
+    return mesh_node
+
+
+def fix_vertex(vtx):
```

Comment:
Probably move this function into utils.py  and sharing with mesh.py

Reply:
fixed